### DISCUSSION TITLE
Change readme to reflect your actual self-host release schedule

### DISCUSSION THREAD

**Discussion by jesperordrup** [plausible/analytics#1861] [Self-Hosted Support] [answered]
I'm currently self-hosting. The release schedule for the self-hosting version is described in README as every half year, but I can see that luckily we get it much more often. Looks like monthly. If that's what you're aiming for then I suggest you update the README because it could have a huge effect on self-host adoption as 6 months between could hold ppl back.

Changelog shows that latest release is 1.4.1
https://github.com/plausible/analytics/blob/master/CHANGELOG.md

**Comment by metmarkosaric** [comment, marked as answer]
thanks @jesperordrup! we plan to stick to the every 6 months or so schedule in the future. if there are some critical issues, there might be a release to fix those, but otherwise twice per year is our goal. you can think of the self hosted release as a long term support release. you can read more on reasons why in this post: https://plausible.io/blog/building-open-source

**Reply by jesperordrup** [reply]
Thanks, @metmarkosaric 

It makes sense if it helps your business model to help balance out cloud and self-host, and I'm all behind that. It's of course important. Just so happens that we can't use the hosted version due to customer requirements. With the current release schedule, I'm worried that we can be stuck with a problem for quite some time. Or waiting for an important feature - for instance, allowing invites without opening the server for everybody to register - which prevents us from moving from test to production.

Maybe a self-hosting license fee is an idea?

However, if it's to spare self-hosting solution owners from the hassle of updating, I think it could be dealt with differently.  First, ppl are not forced to update releases. They could just skip. Secondly, all releases between "big releases" could be "alpha/beta" leaving it up to solution owner to decide.

Half a year is a long time. On the internet, it's an eternity.
